packages/kbn-openapi-generator/src/template_service/templates/zod_schema_item.handlebars

```diff
@@ -25,6 +25,8 @@
     {{~else~}}
       .merge({{> zod_schema_item }})
     {{~/if~}}
+    {{~#if ../nullable}}.nullable(){{/if~}}
```

Comment:
Need to update based on how `nullable: true` interacts with the $ref. [Reference Issue](https://github.com/OAI/OpenAPI-Specification/issues/1368)

Reply:
@michaelolo24 is this something you consider a blocker for this PR?

Reply:
Oh, I updated this, so it was more of a note. Updated the text from `Need` => `Needed` as it was done :)